xclim/indices/_threshold.py

```diff
@@ -717,6 +719,68 @@ def maximum_consecutive_tx_days(
     return group.apply(rl.longest_run, dim="time")
 
 
+@declare_units("[area]", sic="[]", area="[area]", thresh="[]")
+def sea_ice_area(sic, area, thresh="15 pct"):
+    """Return the total sea ice area.
+
+    Sea ice area measures the total sea ice covered area where sea ice concentration is above a threshold,
+    usually set to 15%.
+
+    Parameters
+    ----------
+    sic : xarray.DataArray
+      Sea ice concentration [0,1].
+    area : xarray.DataArray
+      Grid cell area [m²]
+    thresh : str
+      Minimum sea ice concentration for a grid cell to contribute to the sea ice extent.
+
+    Returns
+    -------
+    Sea ice area [m²].
+
+    Notes
+    -----
+    To compute sea ice area over a given region, first mask the sea ice concentration.
+    """
+    thresh = units.Quantity(thresh)
+    t = utils.convert_units_to(thresh, sic)
+    out = xarray.dot(sic.where(sic >= t, 0), area) * thresh.to("dimensionless") / t
```

Comment:
line 748 - Am getting even more confused between extent and area I think.   Why do we need to scale by t?

Reply:
I think I get it now...

Reply:
Calculation makes sense to me with the faq link ...

Reply:
Sea ice concentration is either given as % or a fraction [0,1]. This does the conversion on the output rather than the input. Clarified.